pymc/data.py

```diff
@@ -594,6 +594,11 @@ def Data(
     To set the value of the data container variable, check out
     :func:`pymc.Model.set_data`.
 
+    When making predictions or doing posterior predictive sampling, the shape of the
+    registered data variable will most likely need to be changed.  If you encounter an
+    Aesara shape mismatch error, refer to the documentation for
+    :func:`pymc.model.set_data`.
```

Comment:
```suggestion
    :func:`pymc.set_data`.
```

The function is imported as `pymc.set_data` and since 4.0 we document objects in the same place they should be imported.